### PR TITLE
Change system licensing behavior.

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It solves issue https://github.com/codenvy/enterprise/issues/13

### What issues does this PR fix or reference?
Change licensing behavior for dockerized Codenvy.

@sleshchenko, @tolusha, @olexii4, @ashumilova, @vkuznyetsov: please, review this request.

### CODE REVIEW

**Comment by tolusha** [comment]
Is this PR relevant concerning `docs` module?

**Comment by dmytro-ndp** [comment]
No, it isn't. I am going to fix this request to don't include changes for docs project.

**Comment by dmytro-ndp** [comment]
There is new request where all changes from https://github.com/codenvy/codenvy/tree/license branch have been squashed into the one commit https://github.com/codenvy/codenvy/pull/1428 because of it was too hard to resolve a lot of rebase conflicts.

File: assembly/onpremises-ide-packaging-war-factory/pom.xml [hunk 1]
```diff
@@ -32,13 +32,26 @@
...
+        <dependency>
+            <groupId>com.codenvy.onpremises.wsmaster</groupId>
             <artifactId>codenvy-hosted-sso-client</artifactId>
         </dependency>
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.gwt</groupId>
```
Comment (tolusha) [inline assembly/onpremises-ide-packaging-war-factory/pom.xml]: `<scope>provided<scope>` ?

File: assembly/onpremises-ide-packaging-war-dashboard/pom.xml [hunk 2]
```diff
@@ -28,13 +28,26 @@
...
             <artifactId>codenvy-hosted-commons-j2ee</artifactId>
         </dependency>
         <dependency>
             <groupId>com.codenvy.onpremises.wsmaster</groupId>
             <artifactId>codenvy-hosted-sso-client</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.gwt</groupId>
+            <artifactId>gwt-user</artifactId>
+            <version>${com.google.gwt.version}</version>
```
Comment (tolusha) [inline assembly/onpremises-ide-packaging-war-dashboard/pom.xml]: `<scope>provided</scope>` ?

File: assembly/onpremises-ide-packaging-war-platform-api/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiModule.java [hunk 3]
```diff
@@ -453,5 +454,11 @@ protected void configure() {
         install(new com.codenvy.machine.agent.CodenvyAgentModule());
         bind(org.eclipse.che.api.environment.server.InfrastructureProvisioner.class)
                 .to(com.codenvy.machine.agent.CodenvyInfrastructureProvisioner.class);
+
+        // install system license verification staff
```
Comment (tolusha) [inline assembly/onpremises-ide-packaging-war-platform-api/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiModule.java]: Typo: `staff` -> `stuff`

File: assembly/onpremises-ide-packaging-war-factory/pom.xml (hunk 1 above)
Comment (dmytro-ndp) [inline assembly/onpremises-ide-packaging-war-factory/pom.xml]: I will check it, thanks.
BTW: there is fixed pull request here https://github.com/codenvy/codenvy/pull/1428. Please, comment on there.

Comment (dmytro-ndp) [inline assembly/onpremises-ide-packaging-war-factory/pom.xml]: It works. Thank you for notice.

**Comment by skabashnyuk** [comment]
@tolusha @vkuznyetsov  do we need this pr in master?

**Comment by tolusha** [comment]
We can close it.

**Pull request closed without merging**